challenge.js

```diff
@@ -43,6 +48,10 @@
 
 // ANSWER
 
+document.querySelector('div>p')
```

Comment:
There is some danger to be aware of regarding `document.querySelector`: it selects the *first* matching element. `document.querySelectorAll`, on the other hand, selects *all* matching elements.

Your selector string is good, though. It will select any p that is a child of a div. Here is an explanation and sample solution:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

1